dockerfiles/run/guice/cassandra-ldap/destination/conf/cassandra.properties

```diff
@@ -1,6 +1,9 @@
 # Configuration file for cassandra mailbox
 cassandra.nodes=cassandra
 cassandra.keyspace=apache_james
+#cassandra.user=cassandra
+#cassandra.password=cassandra
+#cassandra.ssl=false
 cassandra.replication.factor=1
```

Comment:
While we are at it, we should explain which parameter can be changed.
Replication factor change has no effect once tables are created for example.

Reply:
I think we made the choice of not documenting things twice, I'd rather write this in the doc and link it from this file

Reply:
See https://github.com/linagora/james-project/pull/2800